crates/egui_demo_lib/src/demo/plot_demo.rs

```diff
@@ -368,7 +375,7 @@ impl MarkerDemo {
         markers_plot
             .show(ui, |plot_ui| {
                 for marker in self.markers() {
-                    plot_ui.points(marker);
+                    plot_ui.owned_points(marker);
```

Comment:
This is a slightly annoying name change, but maybe fine

Reply:
Will be fixed